webapp/src/app/course-welcome/course-welcome.component.css

```
@@ -0,0 +1,11 @@
+:host ::ng-deep .ql-snow {
```

Biome (2.1.2)

[error] Unexpected unknown pseudo-elements: ng-deep

See MDN web docs for more details.
Use a known pseudo-elements instead, such as:


(lint/correctness/noUnknownPseudoElement)